core/lib/types/Cargo.toml

```diff
@@ -21,6 +21,8 @@ zksync_config = { path = "../config" }
 codegen = { git = "https://github.com/matter-labs/solidity_plonk_verifier.git", branch = "dev" }
 zkevm_test_harness = { git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.3.3" }
 zksync_protobuf = { version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "5b3d383d7a65b0fbe2a771fecf4313f5083be9ae" }
+zkevm_test_harness_1_4_1 = { package = "zkevm_test_harness", git = "https://github.com/matter-labs/era-zkevm_test_harness.git", rev = "12f717f12520b2ab87458f125950d7b19e6eb74e" }
```

Comment:
i think there are pros/cons, branches might not be long living and new commits can break existing functionality

Reply:
true, though it seems you specifically wanna import v1.4.1 here - in addition, in case there are any other fixes we need to make to this branch it's easier to just `cargo update` rather than push a PR that changes the revision. in any case, the logic we want in v1.4.1 should work together with this module, so breakage shouldn't be expected. not to mention that we do keep versions locked on branches in crypto repos! so they shouldn't be short lived

Reply:
fair point, we'll have to wait until https://github.com/matter-labs/era-zkevm_test_harness/pull/75 gets merged in then

Reply:
Just in case, you'll anywhere have to create a PR since we commit all the revisions in Cargo.lock